src/styles/main.scss

```diff
@@ -1,3 +1,66 @@
 body {
```

Comment:
Checklist p3. You need to fix it in the whole doc

Reply:
block fixed